codespell_lib/_codespell.py

```diff
@@ -343,29 +386,41 @@ def parse_options(args):
     parser.add_argument('-x', '--exclude-file', type=str, metavar='FILE',
                         help='ignore whole lines that match those '
                              'in the file FILE. The lines in FILE '
-                             'should match the to-be-excluded lines exactly')
-
+                             'should match the to-be-excluded lines exactly',
+                        ).complete = shtab.FILE
+
+    interactive_opts = {
+        -1: '',
```

Comment:
Where has this -1 option come from?

Reply:
Comes from appveyor test, if we don't add it, test will fail.